src/controllers/UserController.js

```diff
@@ -37,6 +37,33 @@ export default class UserController {
     }
   }
 
+  /**
+ * @name getUser
+ * @description Allows user to sign in user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async getUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await UserServices.getUserByEmail(email);
```

Comment:
You don't need to perform this operation here again if you have already added it as a middleware. You can retrieve the user object you assigned a value from the middleware. That would be `const {user} = req`, this would return the user object to you in this controller method

Reply:
It's now imlemented